src/components/block/api.ts

```diff
@@ -81,10 +81,11 @@ function BlockAPI(block: Block): void {
      * @param {string} methodName - method to call
      * @param {object} param - object with parameters
      *
-     * @returns {void}
+     * @returns {any}
      */
-    call(methodName: string, param?: object): void {
-      block.call(methodName, param);
+    // eslint-disable-next-line
```

Comment:
I don't like an idea to ignore this rule. Why not use the `unkown` type?

Reply:
How do I ignore `eslint(jsdoc/no-undefined-types)` for `unknown` type in js-doc comment?

Reply:
Also, there is no issue assigned to this PR. So I just added the link to this PR in the CHANGELOG.md file, is that ok?